common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -2946,7 +2946,11 @@ def robust_usage_key(block_key):
         output_fields = dict(jsonfields)
         for field_name, value in output_fields.iteritems():
             if value:
-                field = xblock_class.fields.get(field_name)
+                field = None
+                try:
+                    field = xblock_class.fields.get(field_name)
+                except AttributeError:
+                    pass
                 if field is None:
                     continue
```

Comment:
This could more simply be 
```
try:
    field = xblock_class.fields.get(field_name)
except AttributeError:
    continue
```

without `field = None` or `if field is None:`.